libmpv/qthelper.hpp

```diff
@@ -97,8 +97,8 @@ static inline QVariant node_to_variant(const mpv_node *node)
 }
 
 struct node_builder {
-    node_builder(const QVariant& v) {
-        set(&node_, v);
+    node_builder(const QVariant& v, bool force_list = false) {
```

Comment:
How does it get coerced to a QString? The node_builder explicitly checks QVariant.type(), so this shouldn't be possible.